playwright-e2e/tests/pancan/enrollment/redirect-glioblastoma-study.spec.ts

```diff
@@ -32,7 +32,14 @@ test.describe('Redirect to Brain cancer project', () => {
     // See a popup redirects
     const modal = new Modal(page);
     await expect(modal.toLocator().locator('.confirm-dialog-title')).toHaveText('Your participation');
-    await expect(modal.toLocator()).toHaveScreenshot('modal-redirect.png');
```

Comment:
switched from toHaveScreenshot to toMatchAriaSnapshot because a bug where the resulting snapshot is stretched and bigger than it should be was occurring. Using toMatchAriaSnapshot seems to have fixed it.